### PR TITLE
improved icon of kubescape in readme

### LINKED ISSUE #689: Improving the Readme
# Describe the bug
 
Currently, the icon in the readme is left aligned including the badge below it which doesn't look good. 
2> It asks for liking but there is no way people can react to that call

# Environment
OS: the OS + version you’re running Kubescape on, e.g Ubuntu 22.04 LTS
Version: the version that Kubescape reports when you run `kubescape version`
```
Your current version is:
```

# Expected behavior

1> It can be centre aligned giving it a better look and feel. 
2> we can ask users to give a ⭐ to the report instead. 

.

### PR DESCRIPTION
**Pull request by legendarykamal**
## Describe your changes

Fixes #689

1> The icon was centre aligned giving it a better look and feel.
2> we have asked users to give a star to the repo .

## Screenshots - If Any (Optional)
Latest Improvement 

image

## Checklist before requesting a review
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<img src="docs/kubescape.png" width="300" alt="logo" align="center">
+<div align="center">
+    <img src="docs/kubescape.png" width="300" alt="logo">
+</div>
```
Comment (vladklokun) [inline README.md:3]: issue: logo is centered, but the badges aren’t

The badges look lonely that way. Let’s add a horizontal rule to visually separate the logo and anchor the badges to the rule.

```suggestion
<div align="center">
    <img src="docs/kubescape.png" width="300" alt="logo">
</div>

---

```

File: README.md [hunk 2]
```diff
@@ -46,7 +48,8 @@ kubescape scan --submit --enable-host-scan --verbose
 
 </br>
 
-### Click [👍](https://github.com/kubescape/kubescape/stargazers) if you want us to continue to develop and improve Kubescape 😀
+### Please Give a [⭐](https://github.com/kubescape/kubescape/stargazers) the repo inorder to show support 
+### and if you want us to continue to develop and improve Kubescape 😀 .
```
Comment (vladklokun) [inline README.md:52]: issue: the heading splits in two, sentence should be grammatically improved

Let’s change it so the header stays put and make the sentence a bit more idiomatic.

```suggestion
### [Star ⭐](https://github.com/kubescape/kubescape/stargazers) the repo if you want us to continue developing and improving Kubescape 😀
```

**Review by vladklokun — CHANGES_REQUESTED** [review]
Good catches! Let’s refine the PR a bit and move to merging it.

Please also note that according to our [contribution guidelines](https://github.com/kubescape/kubescape/blob/f3623dccf65df2ba2006c4b2917aa94d528bcf59/CONTRIBUTING.md) Pull Requests should target the `dev` branch, not the `master`. So please rebase your commits on `dev` before submitting your changes again.

Thanks!

**Comment by legendarykamal** [comment]
Nice suggestions @vladklokun :smile: I will refine it

**Comment by vladklokun** [comment]
> But it did mention that unless they are documentary changes since this was one so i made it against master.

Oh, did it? I can’t find it. Could you point me to where you’ve seen that, please?

**Review by vladklokun — APPROVED** [review]

**Comment by vladklokun** [comment]
Good job, looks good to me :)

We will be merging this shortly.

**Comment by legendarykamal** [comment]
Thanks @vladklokun :fireworks: 



> > But it did mention that unless they are documentary changes since this was one so i made it against master.
> 
> Oh, did it? I can’t find it. Could you point me to where you’ve seen that, please?

Yes, sure look in this 
image

**Comment by vladklokun** [comment]
Oh, it was the PR template, not the contribution guildelines. You’re right. We will have to adjust them accordingly :)

**Pull request merged**